<a href="https://colab.research.google.com/github/wiv33/A-Learning-python/blob/master/install/kaggle.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Install Kaggle

> pip3 install kaggle

In [38]:
!pip install kaggle

In [15]:
!mkdir .kaggle
!mv .kaggle /root/

In [42]:
# !echo '{"username":"name","key":"key"}' > /root/.kaggle/kaggle.json

In [ ]:
# !cat /root/.kaggle/kaggle.json

In [54]:
!chmod 755 /root/.kaggle/

# Download competitions

> kaggle competitions download -c word2vec-nlp-tutorial

In [45]:
!kaggle competitions download -c word2vec-nlp-tutorial

403 - Forbidden


Warning: Looks like you're using an outdated API Version, please consider updating (server 1.5.6 / client 1.5.4)
403 - Forbidden

In [30]:
# !pip3 install --upgrade kaggle

Requirement already up-to-date: kaggle in /usr/local/lib/python3.6/dist-packages (1.5.8)


In [48]:
!pip3 install --upgrade --force-reinstall --no-deps kaggle

Processing /root/.cache/pip/wheels/94/a7/09/68dc83c7c14fdbdf5d3f2b2da5b87e587bfc1e85df69b1130c/kaggle-1.5.8-cp36-none-any.whl
  Found existing installation: kaggle 1.5.8
    Uninstalling kaggle-1.5.8:
      Successfully uninstalled kaggle-1.5.8


In [55]:
!kaggle competitions download -c word2vec-nlp-tutorial

403 - Forbidden
